### PR TITLE
Major cleanup/formatting

### PR DESCRIPTION
**Pull request by jabielecki**
No code changes here, just preparing the field.

The backticks (``) are for sorce code formatting, not for headers.

Fix MD022/blanks-around-headings/blanks-around-headers:
Headings should be surrounded by blank lines

Remove hanging whitespace.

Remove double blank lines.

Format title section.

Do not show users the parts that are not done, but retain them
as comments.

Say "the lastest minor version" instead of "0.12.26" to avoid being
constantly outdated here.

Use lists nested in lists instead of blockquotes (>) because:

- the blockquotes are by convention used only to cite another text,
- the blockquotes render quite flashy.

All the example code had to be indented one level, otherwise
the code wouldn't properly render inside the sublists.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,399 +1,394 @@
...
+* All terraform modules should be versioned using Git tags. Git tag name should adhere to the following naming convention
 
 > Major.Minor.Patch => 0.1.0
 
-* Version should be pinned in the root module where the modules are called from using the `ref` argument 
+* Version should be pinned in the root module where the modules are called from using the `ref` argument:
 
       module "bootstrap_bucket" {
         source          = "spring.paloaltonetworks.com/mekanayake/terraform-aws-panfw-bootstrap?ref=0.1.0"
         bucket_prefix   = "GlobalProtect-bootstrap-"
```
Comment (damianfedeczko) [inline README.md:32]: Is the trailing dash intended in "GlobalProtect-bootstrap-"? Just asking :)

File: README.md [hunk 2]
```diff
@@ -1,399 +1,394 @@
...
 
-      resource "aws_subnet" "in_secondary_cidr" {
-        vpc_id     = vpc_id
-        cidr_block = each.value.cidr_block
-        ipv6_cidr_block = lookup(each.value, "ipv6_cidr_block", null)
-      }
+        resource "aws_subnet" "in_secondary_cidr" {
+          vpc_id     = vpc_id
+          cidr_block = each.value.cidr_block
+          ipv6_cidr_block = lookup(each.value, "ipv6_cidr_block", null)
```
Comment (damianfedeczko) [inline README.md:65]: Nit: fix formatting so the equal signs in the key-value pair are one under another?

File: README.md (hunk 1 above)
Comment (jabielecki) [reply README.md:32]: Not my code, but yes - it is intended.

File: README.md (hunk 2 above)
Comment (jabielecki) [reply README.md:65]: duh, thanks for catching this 👍

**Pull request merged**